templates/web.template.yml

```diff
@@ -12,6 +13,7 @@ params:
 run:
   - exec: /usr/local/bin/ruby -e 'if ENV["HOSTNAME"] == "intercity.example.com"; puts "Aborting! Domain is not configured!"; exit 1; end'
   - exec: /usr/local/bin/ruby -e 'if ENV["SECRET_KEY_BASE"] == "secret_key_base"; puts "Aborting! Secret key base is not configured!"; exit 1; end'
+  - exec: /usr/local/bin/ruby -e 'if ENV["FROM_EMAIL"] == "example@example.com"; puts "Aborting! FROM_EMAIL is not configured"; exit 1; end'
```

Comment:
This ENV var is there by default (I set it in web.template.yml). So only if someones touches code they are not suppose to, it can break